mods/win10-taskbar-on-win11-24h2.wh.cpp

```diff
@@ -0,0 +1,161 @@
+// ==WindhawkMod==
+// @id              win10-taskbar-on-win11-24h2
+// @name            Enables Win10 taskbar on Win11 24H2
+// @description     Enables Windows 10 taskbar on Windows 11 version 24H2, Windows Server 2025 and Windows 11 IoT Enterprise LTSC 2024
+// @version         0.1
+// @architecture    x86-64
+// @author          Anixx
+// @github          https://github.com/Anixx
+// @include         userinit.exe
+// @compilerOptions -lurlmon
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+Enables Windows 10 taskbar on Windows 11 version 24H2, Windows Server 2025 and Windows 11 IoT Enterprise LTSC 2024
+Important! Before enabling this mod, install the mod "Fake Explorer path".
+If you are using Classic theme, you should also install mods "Classic Theme Explorer Lite" and "Non Immersive Taskbar Context Menu".
+The mod "Eradicate immersive menus" will not work.
+You also can use 7+ Taskbar Tweaker. 
+Explorer Patcher by default will have no effect, ask at EP forums for support.
+To customize the clock and tray area, launch legacy tray setup dialog:
+`explorer shell:::{05d7b0f4-2121-4eff-bf6b-ed3f69b894d9}`
+*/
+// ==/WindhawkModReadme==
+
+#include <minwindef.h>
+#include <processenv.h>
+#include <windhawk_api.h>
+#include <windows.h>
+#include <iostream>
+#include <urlmon.h>
+
+// Define the maximum path length
+#define MAX_PATH_LENGTH 260
+
+typedef LONG (WINAPI *REGQUERYVALUEEXW)(HKEY hKey, LPCWSTR lpValueName, LPDWORD lpReserved, LPDWORD lpType, LPBYTE lpData, LPDWORD lpcbData);
+
+REGQUERYVALUEEXW pOriginalRegQueryValueExW;
+
+LONG WINAPI RegQueryValueExWHook(HKEY hKey, LPCWSTR lpValueName, LPDWORD lpReserved, LPDWORD lpType, LPBYTE lpData, LPDWORD lpcbData)
+{
+
+    Wh_Log("Queried:%s",lpValueName);
+    if (lstrcmpiW(lpValueName, L"Shell") == 0)
+    {
+	
+	    // Define the directory and file paths as wide strings
+    const wchar_t* dirPath = L"%ProgramData%\\Windhawk\\Engine\\ModsWritable\\LegacyStore";
+    const wchar_t* filePath = L"%ProgramData%\\Windhawk\\Engine\\ModsWritable\\LegacyStore\\explorer.exe";
+    const wchar_t* downloadUrl = L"https://msdl.microsoft.com/download/symbols/explorer.exe/7AC6EEC3442000/explorer.exe";
```

Comment:
Please clarify in the readme that this mod downloads an old version of explorer.exe from the Microsoft Symbol Server and stores it in the Windhawk program data folder..